silverCrawler/silverCrawler-war/src/main/java/org/silverpeas/components/silvercrawler/control/SilverCrawlerSessionController.java

```diff
@@ -67,17 +66,19 @@
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.Calendar;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.Date;
 import java.util.HashMap;
 import java.util.List;
-import java.util.Locale;
 import java.util.regex.Pattern;
 
 public class SilverCrawlerSessionController extends AbstractComponentSessionController {
 
+  private static final String READ_WRITE_ACTIVATED = "readWriteActivated";
+  private static final String CHECK_RWSETTINGS_ACCESS =
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "CHECK_RWSETTINGS_ACCESS" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1068)